src/applications/claims-status/actions/index.jsx

```diff
@@ -147,16 +148,55 @@ export function fetchClaimsSuccess(response) {
   };
 }
 
-export function getClaimsV2() {
```

Comment:
Will we also be making use of the `show` (`/v0/evss_claims_async/:id`) endpoint mentioned in the linked issue in another PR?